.vscodeignore

```diff
@@ -0,0 +1,11 @@
+.vscode/**
+.vscode-test/**
+out/test/**
+test/**
+src/**
+**/*.map
```

Comment:
What does this do? I don't quite understand this

Reply:
Specifically the line about '.map'? They're generated for every build, but their not necessary in production

https://stackoverflow.com/questions/17493738/what-is-a-typescript-map-file